src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -651,14 +651,26 @@ private void AddIdTokens(Dictionary<string, Func<TokenMatch, string>> tokenHandl
             tokenHandlers["{TvMazeId}"] = m => series.TvMazeId > 0 ? series.TvMazeId.ToString() : string.Empty;
         }
 
-        private void AddPreferredWords(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, Series series, EpisodeFile episodeFile, List<string> preferredWords = null)
+        private void AddPreferredWords(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, Series series, EpisodeFile episodeFile, Dictionary<string, List<string>> profilePreferredWords = null)
```

Comment:
`profilePreferredWords` -> `preferredWords`

Reply:
I reverted the name change up the hierarchy up through the interface.